elfo-logger/src/filtering_layer.rs

```diff
@@ -1,46 +1,98 @@
-use tracing::{subscriber::Interest, Level, Metadata, Subscriber};
-use tracing_subscriber::layer::{Context, Layer};
+use std::sync::Arc;
+
+use arc_swap::ArcSwap;
+use tracing::{metadata::LevelFilter, subscriber::Interest, Metadata, Subscriber};
+use tracing_subscriber::{
+    filter::Targets,
+    layer::{Context, Layer},
+};
 
 use elfo_core::{logging::_priv::CheckResult, scope};
 
 use crate::stats;
 
-#[non_exhaustive]
-pub(crate) struct FilteringLayer {
-    other_max_level: Level,
+struct FilteringConfig {
+    targets: Targets,
+}
+
+impl Default for FilteringConfig {
+    fn default() -> Self {
+        Self {
+            targets: Targets::new().with_default(LevelFilter::INFO),
```

Comment:
I believe it should be `TRACE`, otherwise it overrides actor levels until the logger is configured.